bundle/com.github.lruzicka.Needly.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.lruzicka.Needly.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Needly</name>
+  <summary>openQA needle editor</summary>
+  <description>
+    <p>
+Needly is an openQA needle editor written in Python. It creates or modifies needles for the openQA tests. 
```

Comment:
by reading this I have no clue what it does.

Reply:
Well, this is a highly specific application. When libvirt VM is running, the application can connect to it and take a screenshot from that VM. Then you can draw a rectangle (area) over the screenshot, provide some metadata and save it which creates a "needle", basically a png and a json file. These needles are used for openQA testing. People doing openQA should know what the purpose of the application is, the rest of the world will probably not be interested in it.

Reply:
I'm really not a fan of "if you don't understand then it is not for you". It feels like gatekeeping, and since the files are not coming from upstream, then it's a good occasion to have a much better description.